pyrevolve/config.py

```diff
@@ -91,6 +98,18 @@ def str_to_address(v):
          "Loads a yaml robot."
 )
 
+parser.add_argument(
```

Comment:
the recording plugin got removed, right? It would be worth to check on these parameters and make them easier to use